src/map/lua/lua_baseentity.cpp

```diff
@@ -9924,6 +9924,12 @@ int32 CLuaBaseEntity::checkImbuedItems(lua_State* L)
 inline int32 CLuaBaseEntity::isDualWielding(lua_State* L)
 {
     TPZ_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
+    if (m_PBaseEntity->objtype == TYPE_NPC)
+    {
+        lua_pushboolean(L, false);
+        ShowError("lua::isDualWielding :: NPCs don't wield weapons!\n");
+        return 1;
+    }
 
     CBattleEntity* PBattle = (CBattleEntity*)m_PBaseEntity;
```

Comment:
Reminder for tomorrow about c-style cast thing we discussed in discord, but otherwise :+1: